pages/common/ugrep.md

```diff
@@ -0,0 +1,37 @@
+# ugrep
+
+> Ultra fast search tool with query UI, fuzzy search and more.
```

Comment:
Could we use a non-subjective description here?
```suggestion
> A search tool with an interactive query UI, fuzzy search and more.
```

Reply:
Well, the benefit of ugrep is also its great speed. I know... there are lots that claim speed but ugrep runs multiple threads and we worked hard and continued to do so to beat ripgrep on several fronts. We succeeded to do so. Perhaps "_Ultra fast search tool with interactive query UI, fuzzy search and more._"?

Reply:
Hmmm, perhaps this is less subjective?
```suggestion
> Search tool with a focus on speed. Has an interactive query UI, fuzzy search and more.
```

Reply:
Great suggestion! That works and is not too long.